package.json

```diff
@@ -43,24 +43,34 @@
   "author": "Netlify Inc.",
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.11.4",
-    "@babel/preset-env": "^7.11.5",
-    "@rollup/plugin-babel": "^5.2.0",
-    "@rollup/plugin-commonjs": "^21.0.0",
-    "@rollup/plugin-inject": "^4.0.2",
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-inject": "^4.0.4",
     "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^11.0.0",
-    "@types/node": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@types/node": "^16.11.19",
     "buffer-es6": "^4.9.3",
     "del": "^6.0.0",
-    "make-dir": "^3.1.0",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^2.6.6",
     "path-type": "^4.0.0",
     "process-es6": "^0.11.6",
-    "rollup": "^2.23.1",
+    "rollup": "^2.63.0",
     "rollup-plugin-node-polyfills": "^0.2.1",
     "rollup-plugin-terser": "^7.0.2",
-    "typescript": "^4.1.5"
+    "typescript": "^4.5.4"
```

Comment:
Is typescript really used? It's a big dependency as you probably know, and it's probably intended but I thought I'd ask just to be safe.

Reply:
@erezrokah I tested the package without typescript as a dependency and tests pass., so either TS isn't covered by the current tests or it's an unused dependency...

Reply:
Looks like it was introduced in https://github.com/netlify/netlify-plugin-edge-handlers/commit/1b7ed5e1c7d722bea1cfa25b565274f909c0d763

Doesn't look like we need it

Reply:
The only `ts` reference I can find is https://github.com/netlify/netlify-plugin-edge-handlers/blob/2c5c9a6837995282c89b78c8bccbff928eb18ca2/src/consts.js#L3

Reply:
Regardless, it'd be better to remove typescript in a separate PR, I just noticed it here and thought I'd ask about it.

I'll make a PR after this lands.